recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
+    - icu
     - jom  # [win]
     - jpeg 8*  # [not osx]
-    - libpng >=1.6.21,<1.7  # [not osx]
+    - libpng  # [not osx]
```

Comment:
For some strange reason, `fontconfig` is tied to libpng `1.6.17` so this pin was generating resolution conflicts. Again, I don't have time to do fix this....